src/server/service/kv.rs

```diff
@@ -117,10 +117,11 @@ impl<T: RaftStoreRouter + 'static, E: Engine, L: LockManager> Service<T, E, L> {
 macro_rules! handle_request {
     ($fn_name: ident, $future_name: ident, $req_ty: ident, $resp_ty: ident) => {
         fn $fn_name(&mut self, ctx: RpcContext<'_>, req: $req_ty, sink: UnarySink<$resp_ty>) {
-            let timer = GRPC_MSG_HISTOGRAM_VEC.$fn_name.start_coarse_timer();
+            let timer = Instant::now_coarse();
```

Comment:
Could we give a better name to it because now the type name is no longer a `Timer`.

Reply:
> Could we give a better name to it because now the type name is no longer a `Timer`.

How about `begin_instant`?

Reply:
Sounds good.